README.md

```diff
@@ -25,8 +25,11 @@ To fetch dependencies and get cooking:
 1. `npm install`
 2. As part of the npm install process, the `postinstall` script will install the Bower dependencies for you.
 3. Copy `config/local.json.example` to `config/local.json`, and put your local info in there.
-4. `npm start`
-5. You're up and running! surf to <http://localhost:8080> :surfer:
+4. Run `./server/db/create_db.sh` to create the database, tables, and indexes
```

Comment:
OK, I think this is getting me closer (to go full circle). I was getting this error:
```sh
$ postgres
LOG:  skipping missing configuration file "/usr/local/var/postgres/postgresql.auto.conf"
FATAL:  database files are incompatible with server
DETAIL:  The data directory was initialized by PostgreSQL version 9.3, which is not compatible with this version 9.4.0.
```
This seemed to help me reach database nirvana: https://github.com/Homebrew/homebrew/issues/35240

TL;DR:
```sh
$ rm -rf /usr/local/var/postgres/
$ initdb /usr/local/var/postgres/
```